app/components/directives/settings/ldapsetting.html

```diff
@@ -103,8 +87,25 @@
                                 </div>
                             </div>
                         </div>
-                        <div class="ui row">
 
+                        <div class="ui row">
+                            <div class="ui six wide column">
+                                <div class="field" [ngClass]="{error: service_account_password.errors?.required && formRef.submitted}">
+                                    <label for="service_account_password">Service Account Password</label>
+                                    <input #service_account_password="ngModel"
+                                           type="password"
+                                           id="service_account_password"
+                                           name="service_account_password"
+                                           [(ngModel)]="ldapConfig.service_account_password"
+                                           placeholder="Enter password" required>
+                                    <span class="inlineError" *ngIf="service_account_password.errors?.required && formRef.submitted">
+                                        Please enter ldap service account password
```

Comment:
User is forced to enter service account password whenever he edits ldap settings. I think we shouldn't make it a mandatory field. In User management, when we edit user, password is not mandatory field.